### DISCUSSION TITLE
Selection of Mechanical Actuators Using Simulation-Based Analysis (Project 148)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#14] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #148](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Selection%20of%20Mechanical%20Actuators%20Using%20Simulation-Based%20Analysis)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by auralius** [comment]
Hi, my name is Auralius Manurung, from Universitas Pertamina, Indonesia, and I am going to work on this topic.

**Reply by rhyde005** [reply]
Hi Auralius, great to hear you will work on this topic. I think this is a really interesting research area, and I look forward to hearing how you are getting on. My name is Rick Hyde, and I'm one of the Simscape library authors with a particular interest in mechatronics. You can post questions and thoughts/ideas on this discussion if helpful to you.

**Comment by rudresh-ae** [comment]
Hi Iam Dr. Rudresh M, faculty in aeronautical Engineering. Myself experiencing the innovative projects to take and guiding to students benefit of it.

**Reply by robertogl** [reply]
Hello Dr. Ruresh, 

It is great to hear you are going through the project yourself to better help your students later, very considerate!
Let us know if you need any help.

**Comment by RaghavB31** [comment]
Hello, my name is Raghav Bhatnagar, from the University of Illinois, Urbana Champaign and I am interested in working and collaborating on this project. I am new to MATLAB and am hoping to get some experience working on Simulation-Based Analysis, especially with regards to Mechanical Actuators, so any help or guidance will be greatly appreciated.

**Reply by rhyde005** [reply]
Hi Raghav
Thank you for signing up for this project and for posting on this discussion board. I suggest you start by exploring some of our [mechatronic examples](https://www.mathworks.com/help/sps/examples.html?s_tid=CRUX_topnav), looking under the heading "Robotics and Mechatronics". You might also want to start with this Simscape onramp course [here](https://matlabacademy.mathworks.com/details/simscape-onramp/simscape). Please feel free to post questions related to modeling mechanical actuators here.
best regards
Rick

**Reply by RaghavB31** [reply]
Great, thank you sir.

**Reply by grsasaki** [reply]
Welcome Raghav. As Rick mentioned please do go forward with your ramp-up and post any questions here so that the community can benefit. - Gen

**Reply by RaghavB31** [reply]
Hi, I've been working on the on-ramp simulations to understand MATLAB but I have been using the online version which has started to lag and operate slowly. I am thinking of downloading MATLAB onto my laptop but I am told I do not have a license for it, is there anything that can be done so that I can work more smoothly with the software either online or by downloading it? Thanks

**Reply by rhyde005** [reply]
I have asked our local rep to follow up with you on how best to get access to MATLAB for your project.
kind regards
Rick

**Reply by grsasaki** [reply]
Hi Raghav, I'll follow up with you outside of this thread on your issue, but for the benefit of the other users here, if you run into issues with the performance of our online products such as MATLAB Onramp, you can contact our support team through https://www.mathworks.com/support/contact_us.html.

**Reply by RaghavB31** [reply]
Hi, I've been using the Simulink Onramp and I'm unsure how to do this:
![image](https://github.com/mathworks/MATLAB-Simulink-Challenge-Project-Hub/assets/156875850/3cf8a174-c283-4c4e-b8d2-c74793cd778d)
